docs/csharp/features.md

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
If this document had content, I'd agree with setting up the redirect. I think we can avoid that because it was a placeholder document.